### ISSUE TITLE
(Suggestion) Add quicklink

### ISSUE THREAD

**Issue by Anachron** [picturepan2/spectre#189]
I find it useful to have such a "quicklink" functionality, as [github](https://github.com/picturepan2/spectre#spectrecss) or the [rust api](https://doc.rust-lang.org/std/) uses.

What I mean by this is that if you hover a headline it will show a new element which has the link to the section, so you can easily copy paste it and/or refer to it.

Btw., I just found out about spectre today and I'm impressed. Very good work guys!

**Comment by Anachron** [comment]
I'll try to create this element myself and open a PR once finished.

**Comment by picturepan2** [comment]
@Anachron I already added `anchor` to the Docs https://github.com/picturepan2/spectre/commit/029e72bac88621d5b438135ab43761ec4bd929c8 . Thanks anyway.

**Comment by Anachron** [comment]
Yes but you didn't document the new anchor element, which was what I was talking about ;)

**Comment by Anachron** [comment]
I mean can you add the "anchor" element to the framework itself, not only to the docs.less?

**Comment by picturepan2** [comment]
Yes, I understand. But I am thinking a more common usage of this code.

**Comment by Anachron** [comment]
Something like "Reveal" that lets you reuse the code for any combination? (anchors like on github, display image info (title, subtitle) on hover etc)
